Makefile

```diff
@@ -42,6 +42,16 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+build-contained:
+	if [ -f mender-artifact ]; \
+	then \
+		echo "mender-artifact binary already exists, please move or rename"; \
```

Comment:
Why don't we just remove unconditionally mender-artifact binary?

Reply:
good question... would be in line with the regular `build` task...